### PR TITLE
Add fake antivirus check

### PR DESCRIPTION
**Pull request by suzannehamilton**
Emulate the real antivirus check in a local development environment.

The fake AV saves results to the local API depending on the original name of the file uploaded to.

I expect that we'll add more specific AV results we work on the error pathways.

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version = 1.3.10
\ No newline at end of file
+sbt.version = 1.3.13
```
Comment (TomJKing) [inline project/build.properties:1]: Missing line break at end of file

File: project/plugins.sbt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+resolvers += Resolver.jcenterRepo
+addSbtPlugin("com.frugalmechanic" % "fm-sbt-s3-resolver" % "0.19.0")
```
Comment (TomJKing) [inline project/plugins.sbt:2]: Missing line break at end of file

File: project/build.properties (hunk 1 above)
Comment (suzannehamilton) [reply project/build.properties]: Fixed.

File: project/plugins.sbt (hunk 2 above)
Comment (suzannehamilton) [reply project/plugins.sbt]: Fixed.

**Comment by suzannehamilton** [comment]
@TomJKing I've fixed those, thanks!

**Pull request merged**